services/hidconfigurator.md

```diff
@@ -0,0 +1,53 @@
+# HID Configurator
+
+    identifier: 0x1e5758b5
+    
+A service for configuring how Jacdac device map to HID input events. Users can have multiple configurations and swap between them using the `swapTo` or `next` commands.
+
+## Registers
+
+    rw num_configurations: u16 @ 0x80
```

Comment:
alternatively, we could have a command to return all configurations, and a clear command.